src/core/frameworks/blocknative/Onboard.ts

```diff
@@ -51,14 +52,16 @@ export class BlocknativeWalletImpl implements Wallet {
     const rpcUrl = getNetworkRpc(network);
     const appName = 'Debt DAO';
 
+    const { MAINNET_PROVIDER_HTTPS } = getConstants();
+
     const wallets = [
       {
         walletName: 'metamask',
       },
       {
         walletName: 'walletConnect',
         rpc: {
-          [networkId]: 'https://eth-goerli.public.blastapi.io',
+          [networkId]: MAINNET_PROVIDER_HTTPS,
```

Comment:
why cant we access the provider thats being used inside the wallet?
Or at the very least use dynamic `rpcUrl` we define above

Reply:
good catch - switched to the dynamic `rpcUrl`